### ISSUE TITLE
Testfor problem when installing JedCore

### ISSUE THREAD

**Issue by soupcant95** [plushmonkey/JedCore#14]
Hi. Latest version of both Jedcore and ProjectKorra.

Ever since I installed this awesome plugin on my server, our command block machines have been stopping working and/or been strange. I searched, and the issue was, that a tesfor tag wasn't working. Especifically, this: [Team=]. We tried to see if removing the plugin worked, and it did. We didn't even need to remove PK.

Please fix this, as we like the plugin but the testfor is something we use often on our machines and builds.

**Comment by plushmonkey** [comment]
This is probably related to the bending board since it uses teams to manage the side scoreboard. You could try disabling it in board.yml.

Fixing it would require completely rewriting the entire bending board code. I might do it one day, but it could take some time.

**Comment by soupcant95** [comment]
What am I supossed to disable..? Entyre scoreboard?


Also, wow... So sorry for commenting 6 days late, most people don't respond quick i ntheir issue page... Good to know you respond quick! I will be opening another issue soon, after this one is closed. Thanks!

**Comment by soupcant95** [comment]
Sorry to say, but I tried it right now and it appears to not have worked. I disabld the entyre scoreboard, reloaded the server, and nothing.

Not your fault, you did something good with the plugin, but i you know a fix, please tell me.

**Comment by plushmonkey** [comment]
I'm not sure what else could be causing it because only the bending board uses teams.

I never do anything with command blocks, so I don't know any way to replicate the problem. If you can tell me some kind of minimal setup that would replicate it, I could try to track it down.

**Comment by plushmonkey** [comment]
I just looked into how scoreboards work, and I think I understand why it isn't working.

The testfor command seems to only check the server-controlled scoreboard when the player is set to that scoreboard. If they have their own scoreboard, such as the side scoreboard used for the slots, they won't be found as part of any of the server-controlled teams. Sadly, I couldn't find any easy way around this, so the teams issue can't be fixed while bending boards are enabled.

I pushed a commit that fixes it when the bending board is disabled. It was setting the player to a brand new empty scoreboard when it was disabled instead of back to the main server-controlled one. My understanding of command blocks and scoreboards is pretty limited, so this fix might not work, but it seemed to work for what I tried.

[Here](https://www.dropbox.com/s/xx10vtjwy98trz6/JedCore-2.6.5-Spigot1.12.2-PK1.8.6-boardfix.jar?dl=0) is a compiled version with that change in it.
